CRM/Pledge/Form/Payment.php

```diff
@@ -122,29 +135,24 @@ public function buildQuickForm() {
    * Process the form submission.
    */
   public function postProcess() {
-    // get the submitted form values.
     $formValues = $this->controller->exportValues($this->_name);
-    $params = array();
-    $formValues['scheduled_date'] = CRM_Utils_Date::processDate($formValues['scheduled_date']);
-    $params['scheduled_date'] = CRM_Utils_Date::format($formValues['scheduled_date']);
```

Comment:
This line was using 'schedule_date' (no 'd') in the original

Reply:
*scowls*